changes/626.feature.md

```diff
@@ -0,0 +1 @@
+refactor(webserver): `decrypt_payload()` as a middleware so that applied to `web_handler()` and `login_handler()`
```

Comment:
This format is for commit messages, not news fragments. Please refer changelog/README.md
(https://github.com/lablup/backend.ai/blob/main/changes/README.md#in-the-middle-news-fragments)

Reply:
Furthermore, it is not recommended to use **force** push. Don't worry, all the commits will be squashed and rebased when this PR is merged into main branch.